ee/modules/381-l2-load-balancer/crds/doc-ru-l2-load-balncer.yaml

```diff
@@ -20,6 +20,11 @@ spec:
                   sourceRanges:
```

Comment:
pls, configure the externalTrafficPolicy of L2LoadBalancer in ingress-nginx inlet explicitly.